docs/mkdocs.yml

```diff
@@ -86,8 +86,8 @@ markdown_extensions:
   - pymdownx.inlinehilite
   - pymdownx.tilde
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```

Comment:
Context:

https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#emoji

I don't know when this changed but thanks for handling it (seems our docs still rendered emoji).